frontend/src/app/insight/page.tsx

```diff
@@ -0,0 +1,116 @@
+import "globals";
+import Link from "next/link";
+import { MaterialSymbol } from "react-material-symbols";
+import { Button } from "@/components/ui/button";
+
+export default function Home() {
+  return (
+    <div className="grid h-full flex-1 grid-cols-2">
+      <div className="grid h-full content-center rounded-tr-2xl bg-background">
+        <div className="row-auto grid content-center space-y-12 p-24">
+          <p className="text-6xl font-extrabold">
+            Welcome back, <br></br>John!
+          </p>
+          <p className="text-200 text-200 text-4xl font-medium">
```

Comment:
`text-200` is not a Tailwind CSS classname but a Figma one, you should use `text-secondary-foreground` instead

Reply:
Ups! Thanks for the heads-up!